aries_cloudagent/revocation/tests/test_indy.py

```diff
@@ -154,6 +154,74 @@ async def test_list_issuer_registries(self):
 
         assert len(await self.revoc.list_issuer_registries()) == 2
 
+    async def test_decommision_issuer_registries(self):
```

Comment:
Typo in decommission — missing s

Reply:
d'oh... I don't know how many times I've typed that incorrectly... thanks for the catch.